test/integration/playwright.config.ts

```diff
@@ -15,10 +15,12 @@ const config: PlaywrightTestConfig = {
       name: 'chromium',
       use: { ...devices['Desktop Chrome'] },
     },
-    {
-      name: 'firefox',
-      use: { ...devices['Desktop Firefox'] },
-    },
+    !process.env.SKIP_FF
```

Comment:
Added this for debugging purposes, it was a little bit annoying to have FF executed specs too when I didn't really need them, so it's a small DX optimisation that you can opt-in

Reply:
Can't you just use the flags that Playwright already provides such as `--project=chromium`? Have you tried that?

Reply:
thanks, didn't know about that, it does the trick 👍